test/plugin/scenarios/druid-1.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,313 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+- serviceName: druid-1.x-scenario
+  segmentSize: ge 0
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: Druid/Connection/getConnection
```

Comment:
Could you explain why there are so many `Druid/Connection/getConnection`s?

Reply:
Could you share what is the case of this? This seems very strange to have many local spans, which also impact performance very seriously.

Reply:
Because this method `getConnection` is overloaded in the class `DruidDataSource`, I will fix the issue.